src/main/java/sg/ncl/MainController.java

```diff
@@ -179,6 +179,12 @@ public String jointeam() {
         return "jointeam";
     }
 
+
+    @RequestMapping("/Error_OpenStack")
+    public String Error_OpenStack() {
```

Comment:
Pls change "/Error_OpenStack" to "/error_openstack", change Error_OpenStack() to errorOpenStack()